nsqadmin/logger.go

```diff
@@ -1,5 +1,58 @@
 package nsqadmin
 
+import (
+	"fmt"
+	"strings"
+)
+
 type Logger interface {
 	Output(maxdepth int, s string) error
 }
+
+const (
+	LOG_DEBUG = 1
+	LOG_INFO  = 2
+	LOG_WARN  = 3
+	LOG_ERROR = 4
+	LOG_FATAL = 5
+)
+
+func (n *NSQAdmin) logLevelFromString(level string) int {
+	// check log-level is valid and translate to int
+	switch strings.ToLower(level) {
+	case "debug":
+		return LOG_DEBUG
+	case "info":
+		return LOG_INFO
+	case "warn":
+		return LOG_WARN
+	case "error":
+		return LOG_ERROR
+	case "fatal":
+		return LOG_FATAL
+	case "":
+		return LOG_INFO
```

Comment:
I don't think you need to handle the blank case, this can be considered invalid.

I also don't think you need `ToLower()`, you can just require the lowercase strings. This is not a big deal though.